convert.py

```diff
@@ -0,0 +1,108 @@
+from xml.etree import ElementTree as ET
+import json
+import sys
+
+orig_stdout = sys.stdout
+f = open('output.txt', 'w')
+sys.stdout = f
+
+with open("batch_response_data.json") as json_file:
```

Comment:
This input file is just for testing your code. You should separate the code to render the report from the test data. You can use the test data in your test case.